components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -1403,8 +1433,9 @@ - (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
 
   UIScrollView *oldTrackingScrollView = _trackingScrollView;
 
+  CGFloat stashedHeight = self.bounds.size.height;
```

Comment:
_Tip:_ CGRectGetHeight()

Reply:
+1

CGRectGetHeight() should never be a Tip. Please fix! :)